xml/System/DateTime.xml

```diff
@@ -3434,7 +3436,7 @@ juillet 2009
 ## Remarks  
  The value of this constant is equivalent to 23:59:59.9999999 UTC, December 31, 9999 in the Gregorian calendar, exactly one 100-nanosecond tick before 00:00:00 UTC, January 1, 10000.  
   
- Some calendars, such as the <xref:System.Globalization.UmAlQuraCalendar>, support an upper date range that is earlier than <xref:System.DateTime.MaxValue>. In these cases, trying to access <xref:System.DateTime.MaxValue> in variable assignments or formatting and parsing operations can throw an <xref:System.ArgumentOutOfRangeException>. Rather than retrieving the value of <xref:System.DateTime.MaxValue?displayProperty=nameWithType>, you can retrieve the value of the specified culture's latest valid date value from the `System.Globalization.CultureInfo.DateTimeFormat.Calendar.MaxSupportedDateTime` property.  
+ Some calendars, such as the <xref:System.Globalization.ThaiBuddhistCalendar>, support an upper date range that is earlier than <xref:System.DateTime.MaxValue>. In these cases, trying to access <xref:System.DateTime.MaxValue> in variable assignments or formatting and parsing operations can throw an <xref:System.ArgumentOutOfRangeException>. Rather than retrieving the value of <xref:System.DateTime.MaxValue?displayProperty=nameWithType>, you can retrieve the value of the specified culture's latest valid date value from the `System.Globalization.CultureInfo.DateTimeFormat.Calendar.MaxSupportedDateTime` property.  
```

Comment:
What is `System.Globalization.CultureInfo.DateTimeFormat.Calendar.MaxSupportedDateTime` supposed to mean? It's not referring just to the property, but it's not a valid expression either. I think this should just use an xref: `<xref:System.Globalization.Calendar.MaxSupportedDateTime>`.